### PR TITLE
Catch SDK init client errors and update SDK to 7.7.0

### PR DESCRIPTION
**Pull request by shapkarin**

### CODE REVIEW

**Review by mike-gusev-ae — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   "author": "",
   "readmeFilename": "README.md",
   "dependencies": {
-    "@aeternity/aepp-sdk": "^7.4.1",
+    "@aeternity/aepp-sdk": "^7.6.0",
```
Comment (mradkov) [inline package.json]: Maybe you can try 7.7.0 instead which will be released in the next couple of hours? 🤔

Comment (shapkarin) [reply package.json]: @mradkov nice

Comment (shapkarin) [reply package.json]: thanks a lot for the info, it's released https://github.com/aeternity/aepp-sdk-js/releases/tag/7.7.0 and I updated it to 7.7.0

File: react/client.js [hunk 2]
```diff
@@ -9,33 +9,31 @@ const NODE_URL = 'https://mainnet.aeternity.io';
...
+                    url: TESTNET_URL
+                })
+            } ],
+            compilerUrl: COMPILER_URL
+        });
+    } catch (sdkError) {
+        console.log({ sdkError });
     }
 
     return;
```
Comment (venimus) [inline react/client.js]:
```suggestion
```

**Review by venimus — APPROVED** [review]

**Pull request merged**